group/assignment5/find_all_alphabets.py

```diff
@@ -0,0 +1,38 @@
+from collections import defaultdict
+from assignment5.graph import *
+from assignment5.find_alphabet import build_graph
+
+def find_all_alphabets(dictionary):
+    graph = build_graph(dictionary)
+    letters = graph.vertices
+    letters_to_add = list()
+    added_parents = defaultdict(int)
+    all_alphabets = list()
+    for l in letters:
+        if len(graph.parents[l]) == 0:
```

Comment:
@aliskin  maybe to replace this string with "if not len(graph.parents[l])" will be more pythonic

Reply:
Will it? I think it will be much harder to read and understand.

Reply:
@aliskin Ok, I think it is subjective assumption)